Week04/lesson08/homework/temp_converter/js/index.js

```diff
@@ -0,0 +1,6 @@
+var f;
+var c = 20;
+
+//Do stuff
```

Comment:
So I'm thinking the goal for the first "version" of this assignment was maybe TOO obvious, because most people seemed to have been thinking on a grander scale (including you) from the get-go.

All I wanted for "Version 1" was this:

```
var c = 20;
var f = c * 9/5 + 32;

console.log(f + "°F");
```

... that's it! I just wanted to get you guys comfortable with manipulating the `c` variable and then presenting it back via the `console` (since that's a bit easier than writing back to the HTML).

For "Version 2" you've done the HTML basically perfectly. Consider this helper code:

```
function convert() {
  var c = code_to_get_value_goes_here;    /*get the value of the <input id="c"> from the HTML*/
  var f = c * 9/5 + 32;                   /*this formula was in the slides*/

  document.querySelector("#result").innerHTML = "Result: " + f + "°F";
}

document.querySelector('#convert').onclick = convert;
```

... see if you can work out what belongs in place of `code_to_get_value_goes_here`